tests/v3/specs/petstore.yaml

```diff
@@ -1,10 +1,9 @@
 openapi: 3.0.1
 info:
   title: Swagger Petstore
-  description:
-    "This is a sample server Petstore server.  You can find out more about     Swagger
+  description: 'This is a sample server Petstore server.  You can find out more about     Swagger
```

Comment:
The Swagger Petstore schema was updated because it was technically invalid before. We had modified it to try and test for `schema.responses`, but it was modified into an invalid format. The original Petstore schema was restored, and we now test for `schema.responses` in a better way in our unit tests.